mountpoint-s3/src/cli.rs

```diff
@@ -160,7 +160,7 @@ pub struct CliArgs {
         long,
         help = "Part size for multi-part GET and PUT",
         default_value = "8388608",
-        value_parser = value_parser!(u64).range(1..),
+        value_parser = value_parser!(u64).range(1..usize::MAX as u64),
```

Comment:
I thought this upper bound was already implicit, does it actually change anything?

Reply:
on 64-bit it doesn't change anything, the limit is already there:
```
$ mount-s3 --version
mount-s3 1.6.0
$ mount-s3 <bucket> <path> --debug --foreground --part-size=18446744073709551616
error: invalid value '18446744073709551616' for '--part-size <PART_SIZE>': number too large to fit in target type
```

but in the context of 32-bit system, where `usize::MAX` (type expected by the `S3CrtClient`) is less than `u64::MAX` there will be a change, since the cli argument type is `u64` (and there is no good way to specify `usize`, `value_parser!` macro does not compile with `usize`)

Reply:
Ah, I missed that it was `usize::MAX` not `u64::MAX`!